src/base/preferences.cpp

```diff
@@ -449,6 +451,36 @@ void Preferences::setWebUiLocalAuthEnabled(bool enabled)
     setValue("Preferences/WebUI/LocalHostAuth", enabled);
 }
 
+bool Preferences::isWebUiAuthIPBypassEnabled() const
+{
+    return value("Preferences/WebUI/EnableAuthIPBypass", false).toBool();
+}
+
+void Preferences::setWebUiAuthIPBypassEnabled(bool enabled)
+{
+    setValue("Preferences/WebUI/EnableAuthIPBypass", enabled);
+}
+
+QList<QPair<QHostAddress, int> > Preferences::getWebUiAuthBypassSubnets() const
+{
+    QList<QPair<QHostAddress, int> > subnets;
+    QStringList rawSubnets = value("Preferences/WebUI/AuthBypassIP").toStringList();
```

Comment:
Also this variable is redundant. You can use value("Preferences/WebUI/AuthBypassIP").toStringList() in loop condition directly.